directFeatures.py

```diff
@@ -52,11 +50,16 @@ def match_to_faces(list_face_encodings, list_face_locations, people, resized_ima
             next_unknown += 1
             current = people[name]
             current['face_vec'] = face_encoding
-            current['video_name'] = filename
-            current['video_hash'] = file_hash
+            current['file_name'] = filename
+            current['file_name_hash'] = hashlib.md5(str(filename).encode('utf-8')).hexdigest()
```

Comment:
hoist the hashlib.md5(...)
above line 29 as it does not need to be computed for every face, only the 1st face.